src/main/java/nz/cri/gns/NZSHM22/opensha/enumTreeBranches/NZSHM22_DeformationModel.java

```diff
@@ -28,6 +28,16 @@
             "any",
             null),
 
+    GEOD_NO_PRIOR_2022_UnVwdHVyZUdlbmVyYXRpb25UYXNrOjc5OTBvWWZMVw(
```

Comment:
```
GEOD_NO_PRIOR_2022_RmlsZToyMjE4My4wUGVpWGE=(
            "geodetic, no geological prior constraint, 2022",
            "rupture set RmlsZToyMjE4My4wUGVpWGE=",
 ...
```

@voj Kaj picked the task ID rather than the rupset ID here, so I've correctedd that .  The final `=` is part of the ID, but may not be valid in java , so just sub in `_` in that case.